NuRadioMC/utilities/attenuation.py

```diff
@@ -123,6 +133,23 @@ def get_attenuation_length(z, frequency, model):
             att_length_f[att_length_f < min_length] = min_length
         return att_length_f
 
+    if model == 'GL3':
+        slope_interpolation = scipy.interpolate.interp1d(
```

Comment:
don't you want to move the interpolation ll 137-148 also to the start of the module?

Reply:
It doesn't slow down the code too much since it is just a simple linear interpolation, but I guess it can't hurt. Done